vendor/init.bat

```diff
@@ -165,3 +182,12 @@ if exist "%CMDER_ROOT%\config\user-profile.cmd" (
     echo.
     ) > "%CMDER_ROOT%\config\user-profile.cmd"
 )
+
+exit /b
+
+::
+:: sub-routines below here
+::
+:verbose-output
```

Comment:
I don't care what it's called I just did like a way to easily enable/disable some additional info that could help in troubleshooting.